recipes/r-lasr/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+export DISABLE_AUTOBREW=1
+export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
```

Comment:
I see at https://github.com/r-lidar/lasR/blob/8c789d3f99ac3de6d4d0149855c7517aaa9750de/src/Makevars.in#L2 that `PKG_CXXFLAGS` is used, could you try this instead? Just because I don't see `-D_LIBCPP_DISABLE_AVAILABILITY` in the logs, and found some other packages https://github.com/search?q=org%3Aconda-forge++LIBCPP_DISABLE_AVAILABILITY+language%3AShell+%24%7BR%7D+CMD+INSTALL&type=code using sed replace.

```suggestion
export PKG_CXXFLAGS="${PKG_CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
```

Reply:
thanks I'll try that